### ISSUE TITLE
Quake III Team Arena (2350)

### ISSUE THREAD

**Issue by ipkpjersi** [ValveSoftware/Proton#5160]
# Compatibility Report
- Name of the game with compatibility issues: Quake III Team Arena
- Steam AppID of the game: 2350

## System Information
- GPU: GTX 1070
- Driver/LLVM version: NVIDIA 460.91.03 stable
- Kernel version: 5.4.0-84 LTS on Ubuntu 20.04 LTS
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/ipkpjersi/a565131d0eff7bf82099e40b783f2a9e
- Proton version: 6.3-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
The game attempts to launch and reports Failed to find Steam even though I am launching it from Steam.

## Reproduction
1. Attempt to run Quake III Team Arena with Proton 6.3-6 via Steam
2. Observe that it says Failed to find Steam 

You can find some logs here: https://gist.github.com/ipkpjersi/b3804c20c808ada91301ec2b83690245

[steam-2350.log](https://github.com/ValveSoftware/Proton/files/7174232/steam-2350.log)

Also, I have noticed that this game has Gold status on ProtonDB (https://www.protondb.com/app/2350), so perhaps there is something wrong with my setup?

**Comment by McMarius11** [comment]
on my manjaro amd laptop it runs ootb with 6.3-6

this error looks interesting:
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and h

**Comment by kisak-valve** [comment]
We've seen the Authentication Rejected log message (apparently harmlessly) coming from some part of Steam. Probably unrelated to Proton in general.

**Comment by h1z1** [comment]
Unless that log got eaten by github somehow, those lines look truncated no?  Guessing it failed before then.

**Comment by ipkpjersi** [comment]
Logs read fine to me, I don't believe anything is truncated.

**Comment by h1z1** [comment]
o_O

![troz](https://user-images.githubusercontent.com/17220053/144723297-d3278552-719e-41c3-a396-6bfbba539fb7.png)

Non-potato edit to make it clear these are two different screenshots and taken in case it's a font thing with Github.    Though the raw version also shows it.

![troz2](https://user-images.githubusercontent.com/17220053/144723314-77589583-5834-4a95-ac9b-fa467ff87077.png)


From what I gather it's coming from Steam or Qt?  Kind of expected Xlib but the only real references I could find weren't related.    This bug was the second result.

**Comment by ipkpjersi** [comment]
Yeah, that's in the log files themselves, it sounds like those messages are harmless (it's the generic Steam logs, not the Proton game logs) even though there's some weird encoding going on there I suppose.
